doc/howto/storage_configure_bucket.md

```diff
@@ -20,9 +20,13 @@ Use the following command to delete a storage bucket and its keys:
 
 ## How to configure storage bucket keys
 
+Use the following command to add a bucket key (see {ref}`storage_create_bucket_keys` for more information):
```

Comment:
I'm quite happy with having this here, but I'd intentionally left it out originally to keep inline with the existing structure of not showing how to "create" entities in the "configure" howto (for example pools or volumes howtos). Instead I covered it in more detail here https://linuxcontainers.org/lxd/docs/master/howto/storage_create_bucket/#create-storage-bucket-keys

So perhaps a link/note here to https://linuxcontainers.org/lxd/docs/master/howto/storage_create_bucket/#create-storage-bucket-keys so that readers can find out how to create keys without duplicating content?

Or perhaps (now we have the entity type top-level menu pages you've added in this PR), are we perhaps moving to a "Work with ..." single page for each entity type where the full lifecycle of each entity can be demonstrated in a single page (using sub headings) rather than separate smaller howtos?

Reply:
The only part we're duplicating is the command (the reference goes to the section you're mentioning). I think that is okay, but I was quite confused at first why we only say how to change an existing key here, and not how to create it.
The page is about configuring buckets, and imo, bucket keys are a "sub-concept" of buckets. So maybe the whole "Create storage bucket keys" should be moved to the configuration page?

Reply:
I like the idea of putting all how-tos for a section on one page, but I'm a bit concerned that the volume page will get quite long. What do you think?

Reply:
> The page is about configuring buckets, and imo, bucket keys are a "sub-concept" of buckets. So maybe the whole "Create storage bucket keys" should be moved to the configuration page?

Yeah I see what you mean. My interpretation was that the "configuring" page was only for configuring existing entities (and their sub entities). So different interpretations.

> I like the idea of putting all how-tos for a section on one page, but I'm a bit concerned that the volume page will get quite long. What do you think?

Great! I think we could compromise with Volumes and leave the move/copy and backup howtos separate pages (as they aren't really to do with the lifecycle of a volume itself). Then have a "Manage X" section for each entity where we cover in a single page: create, configuring, viewing (list/get), using (attaching to instances for example) and delete.

Reply:
> Great! I think we could compromise with Volumes and leave the move/copy and backup howtos separate pages (as they aren't really to do with the lifecycle of a volume itself). Then have a "Manage X" section for each entity where we cover in a single page: create, configuring, viewing (list/get), using (attaching to instances for example) and delete.

:laughing: Nice, I was thinking exactly the same, but couldn't come up with a justification on keeping those two pages separate. Let me update that.

Reply:
Updated now.